contents/handbook/engineering/deployment-strategy.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Deployment Strategy
+sidebar: Handbook
+showTitle: true
+---
+
+The way we deploy is a key _product_ decision.
+
+* 'Cloud' = PostHog Cloud, we host it for you in *PostHog's* AWS
```

Comment:
I wonder if it's possible to avoid the word "host" and mentions to infrastructure on Cloud stuff? Not quite sure it is, but just to really have 0 confusion.